backend/app/Http/Controllers/QuestionController.php

```diff
@@ -35,9 +35,21 @@ public function create()
      * @param  \App\Http\Requests\StoreQuestionRequest  $request
      * @return \Illuminate\Http\Response
      */
-    public function store(Request $request)
+    public function store(Category $category, Request $request)
     {
-        //
+        $question_data = $request->validate([
+            'question' => ['required', 'string'],
+            'image' => ['required', 'file'],
+        ]);
+
+        $question = Question::create([
+            'category_id' => $category->id,
+            'question' => $question_data['question'],
+            'slug' => $this->createSlug($question_data['question']),
```

Comment:
You can use [Str::slug](https://laravel.com/docs/8.x/helpers#method-str-slug) method from laravel